### ISSUE TITLE
-Wduplicate-decl-specifier in sound/soc/codecs/jz4725b.c

### ISSUE THREAD

**Issue by nathanchance** [ClangBuiltLinux/linux#354]
```
sound/soc/codecs/jz4725b.c:177:14: warning: duplicate 'const' declaration specifier [-Wduplicate-decl-specifier]
static const SOC_VALUE_ENUM_SINGLE_DECL(jz4725b_codec_adc_src_enum,
             ^
include/sound/soc.h:356:2: note: expanded from macro 'SOC_VALUE_ENUM_SINGLE_DECL'
        SOC_VALUE_ENUM_DOUBLE_DECL(name, xreg, xshift, xshift, xmask, xtexts, xvalues)
        ^
include/sound/soc.h:353:2: note: expanded from macro 'SOC_VALUE_ENUM_DOUBLE_DECL'
        const struct soc_enum name = SOC_VALUE_ENUM_DOUBLE(xreg, xshift_l, xshift_r, xmask, \
        ^
```

Patch submitted: https://lore.kernel.org/lkml/20190208045918.16799-1-natechancellor@gmail.com/

**Comment by nathanchance** [comment]
Patch accepted: https://git.kernel.org/pub/scm/linux/kernel/git/broonie/sound.git/commit/?id=d9186330c881468722c005ee772ca745c81a29ef

**Comment by nathanchance** [comment]
Fixed as of `next-20190211`: https://git.kernel.org/next/linux-next/c/d9186330c881468722c005ee772ca745c81a29ef
